docs/en/framework/ui/angular/quick-start.md

```diff
@@ -25,21 +25,21 @@ You have multiple options to initiate a new Angular project that works with ABP:
 
 ### 1. Using ABP CLI
 
-ABP CLI is probably the most convenient and flexible way to initiate an ABP solution with an Angular frontend. Simply [install the ABP CLI](../../CLI.md) and run the following command in your terminal:
+ABP CLI is probably the most convenient and flexible way to initiate an ABP solution with an Angular frontend. Simply [install the ABP CLI](../../../cli) and run the following command in your terminal:
 
 ```shell
 abp new MyCompanyName.MyProjectName -csf -u angular
 ```
 
-> To see further options in the CLI, please visit the [CLI manual](../../CLI.md).
+> To see further options in the CLI, please visit the [CLI manual](../../../cli).
 
-This command will prepare a solution with an Angular and a .NET Core project in it. Please visit [Getting Started section](../../Getting-Started.md?UI=NG&DB=EF&Tiered=No#abp-cli-commands-options) for further instructions on how to set up the backend of your solution.
+This command will prepare a solution with an Angular and a .NET Core project in it. Please visit [Getting Started section](../../../_deleted/getting-started.md?UI=NG&DB=EF&Tiered=No#abp-cli-commands-options) for further instructions on how to set up the backend of your solution.
```

Comment:
There are still some links here that need to be changed

<img width="789" alt="image" src="https://github.com/abpframework/abp/assets/16813853/28f56d7c-64dd-4455-840b-5651df91286f">

Reply:
I made the necessary change